pandas/_libs/tslibs/parsing.pyx

```diff
@@ -85,8 +85,9 @@ _DEFAULT_DATETIME = datetime(1, 1, 1).replace(hour=0, minute=0,
                                               second=0, microsecond=0)
 
 PARSING_WARNING_MSG = (
-    "Parsing '{date_string}' in {format} format. Provide format "
-    "or specify infer_datetime_format=True for consistent parsing."
+    "Parsing dates in {format} format when dayfirst={dayfirst} was specified. "
```

Comment:
Might still be helpful to show the `date_string` example that would be parsed by the `format`? Or did you think it would be confusing?

Reply:
It might be helpful, but then extra complexity would need to be introduced to ensure that the warning only be emitted once

If the date isn't shown, the warning message is always the same and so is only shown once

Reply:
Okay minimizing the warning occurrence sounds like a good enough reason to me then. Then below in the `format` call it appears `date_string` is still being passed.